testing/testing.rb

```diff
@@ -0,0 +1,49 @@
+require "test/unit"
+require './modules/web_framework.rb'
+
+
+class Testing < Test::Unit::TestCase
+
+  def test_dropdown_options
+    driver = Web_framework.new
+    url = 'file:///form/form.html'
+    driver.get_webpage(url)
+    dropdown = driver.find_element({:id=>'language'})
+    assert_equal(true,driver.check_dropdown_options(dropdown,"english"))
+  end
+  def test_dropdown_item_selection
+    driver = Web_framework.new
+    driver.get_webpage('file:///form/form.html')
+    dropdown = driver.find_element({:id=>'language'})
+    assert_equal('English',driver.select_item_from_dopdown(dropdown,1).text)
+  end
+  def test_find_element
+    driver = Web_framework.new
+    driver.get_webpage('file:///form/form.html')
+    assert_not_equal(nil,driver.find_element({:id=>'language'}))
+  end
+  def test_find_elements
+    driver = Web_framework.new
+    driver.get_webpage('file:///form/form.html')
+    assert_not_equal(nil,driver.find_elements({:tag_name=>'options'}))
+  end
+  def test_element_click
```

Comment:
A better test name would be `test_element_click_when_element_not_found`, since it tests the negative test case.

Reply:
Similarly, write a positive test case this method